README.md

```diff
@@ -17,6 +17,10 @@ What does it mean for an HTML tag to be "semantic"? Give an example of a short s
 
 ```text
 Your answer here.
+An HTML tag is semantic when it describes the type of content it contains.
+ Ex. of a semantic tag: <h1> My Website </h1>
+ Ex. of a non-semantic tag: <div> Part 1 </div>
+
 ```
 
```

Comment:
Yes, this is true. It is describing the function of the content rather than the style of it. Another example might be using `<strong> `which is semantic over `<b>` which is non-semantic